server/apps/archive/archive.py

```diff
@@ -362,17 +367,15 @@ def _duplicate_item(self, original_doc):
         del new_doc[config.ID_FIELD]
         del new_doc['guid']
 
+        new_doc[ITEM_OPERATION] = ITEM_DUPLICATE
         item_model = get_model(ItemModel)
 
         on_duplicate_item(new_doc)
-        item_model.create([new_doc])
-        self._duplicate_versions(original_doc['guid'], new_doc)
-
+        resolve_document_version(new_doc, 'archive', 'PATCH', new_doc)
```

Comment:
There will be no inconsistency: the duplicated article will have version 4 and 4 documents in the arcticle_versions collection (_duplicate_versions adds this version). The duplicated article needs an extra version in order to record the operation.

"We need not to increment the version when the article state is 'Submitted'. "
This is done, see above.